cmake/compilers.cmake

```diff
@@ -56,10 +57,12 @@ function(ADJUST_COMPILER_WARNINGS)
                 -Wno-deprecated-declarations
         )
     else() # GCC-12 or higher
+        message(STATUS "Adjusting compiler warnings for GCC")
         target_compile_options(
             compiler_warnings
             INTERFACE
                 -Wno-deprecated
+                -Wno-deprecated-declarations
```

Comment:
required as I introduce a first case of `[[deprecated]]`

Reply:
🙌